src/client/Licit.js

```diff
@@ -615,6 +618,26 @@ class Licit extends React.Component<any, any> {
     view.dispatch(tr.setSelection(TextSelection.atEnd(view.state.doc)).scrollIntoView());
     view.focus();
   };
+  //To insert empty text property for content without text attribute
+  insertTextNodes = (docJson) => {
```

Comment:
This name is misleading. It's inserting text into nodes, not adding nodes. Also, this isn't recursive so I don't think it is handling nested nodes.